public/app/features/live/dashboard/DashboardChangedModal.tsx

```diff
@@ -36,7 +36,7 @@ export function DashboardChangedModal({ onDismiss, event }: Props) {
       className={styles.modal}
     >
       <div className={styles.description}>
-        The dashboad has been updated by another session. Do you want to continue editing or discard your local changes?
+        The dashboard has been updated by another session. Do you want to continue editing or discard your local changes?
```

Comment:
@Erisa thanks for the contribution! it looks like [lint is failing](https://drone.grafana.net/grafana/grafana/172263/2/5). could you please try to run `yarn run prettier:write` and commit the result?

Reply:
Done! Good catch - looks like the linter thinks this one extra character makes the line too large and wanted to wrap it.